src/coreComponents/physicsSolvers/LinearSolverParameters.cpp

```diff
@@ -40,6 +40,11 @@ LinearSolverParametersInput::LinearSolverParametersInput( std::string const & na
     setInputFlag( InputFlags::OPTIONAL )->
     setDescription( "Preconditioner type. Available options are:\n* " + EnumStrings< LinearSolverParameters::PreconditionerType >::concat( "\n* " ) );
 
+  registerWrapper( viewKeyStruct::dofsPerNodeString, &m_parameters.dofsPerNode )->
+    setApplyDefaultValue( m_parameters.dofsPerNode )->
+    setInputFlag( InputFlags::OPTIONAL )->
+    setDescription( "Dofs per node (or support location) for non-scalar problems" );
```

Comment:
I really do not. I agree with you, but I put it there because it is declared in [LinearSolverParameters.hpp](https://github.com/GEOSX/GEOSX/blob/feature/af1990/contactMechanicsImprov/src/coreComponents/linearAlgebra/utilities/LinearSolverParameters.hpp#L67).
https://github.com/GEOSX/GEOSX/blob/3e355b89c2a51fc987ee1c35ca0ef35f925c324b/src/coreComponents/linearAlgebra/utilities/LinearSolverParameters.hpp#L67

Reply:
Ok, we have a few parameters (like `isSymmetric` and `dofsPerNode`) that are meant to be set by the physics solver only, so unless you have a case where setting this from the input would be of benefit, let's remove the wrapper registration.

Reply:
Wrapper registration removed.